docker-entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
Missed this on my first pass; can we get a `set -e` here (and of course replicated)?